.golangci.yml

```diff
@@ -233,7 +234,7 @@ linters:
     # Additional
     # - lll
     - godox
-    #- gomnd
+    # - mnd
```

Comment:
still remains commented. What is this change.

Reply:
The `gomnd` (magic number) linter does not exist anymore. Now, its name is `mnd`. This linter is about defining constant values/variables (e.g., const TIMEOUT = 2) and then use the `TIMEOUT` in the code. This is something that needs to be done but I cannot do it at the moment and should be done in a future PR.